app/components/DataCardInput.tsx

```diff
@@ -49,11 +49,13 @@ const DataCardInput = ({
   const [editValue, setEditValue] = useState<string>(initialValue);
 
   const isDirty = editValue !== initialValue;
+  const isValid = editValue.trim() !== "";
```

Comment:
hmm, i'm i see how we have unfinished work around input validation here :/ 

i like this, but it is going to break other fields that are nullable/clearable, like the address fields. i think we should break this off into a separate issue to add support for required fields? i know that means this will ship with an unhandled failure state :/

Reply:
👍 
* https://github.com/cityofaustin/atd-data-tech/issues/20878